### DISCUSSION TITLE
Other Layouts in mindmap?

### DISCUSSION THREAD

**Discussion by Jeff-Tian** [mermaid-js/mermaid#4850] [Q&A]
Is mermaid mindmap supporting this kind of [layout](https://www.plantuml.com/plantuml/uml/JOzD3e8m44Rtd6BMtNW192IM5I29HEDsAbKdeLD2MvNRInjDMCsRlFd9LpgFipV4Wy4f4o2r8kHC23Yhm3wi9A0X3XzeYNrgwx1H6wvb1KTjqtRJoYhMtexBSAqJUescwoEUq4tn3xp9Fm7XfUS5HiiFO3Gw7SjT4QUCkkKxLy2-WAvl3rkrtEclBdOCXcnMwZN7ByiN)?

<img width="1280" alt="image" src="https://github.com/mermaid-js/mermaid/assets/3367820/52025046-d1be-4a2b-b236-8726cae7a02d">

**Comment by sudhirkhanger** [comment]
This is super important in my opinion because mindmaps become difficult to use when number of items increases. The items start overlapping.

```mermaid
mindmap
  root((mindmap))
    Origins
      Long history
      ::icon(fa fa-book)
      Popularisation
        British popular psychology author Tony Buzan
    Research
      On effectivness<br/>and features
      On Automatic creation
        Uses
            Creative techniques
            Strategic planning
            Argument mapping
            Creative techniques
            Strategic planning
            Argument mapping
            Creative techniques
            Strategic planning
            Argument mapping
            Creative techniques
            Strategic planning
            Argument mapping
            Creative techniques
            Strategic planning
            Argument mapping
    Tools
      Pen and paper
      Mermaid
```

**Comment by Piratepie** [comment]
There's currently no way to change the structure of the mindmap in mermaid as per the documentation.

It would be nice to see the mind map diagram expand, as the issue @sudhirkhanger raised is valid.

Personally I feel like the mermaid mind map is great up till roughly 20 elements, spread evenly.

Above that, and what might be useful for you, I use flowcharts instead. It's not entirely what they're designed for, but they work really well imo.
Here's an example:

```mermaid
flowchart LR
    root([Debian])
    A([Ubuntu])
    AA([Linux Mint])
    AB([Kubuntu])
    B([LMDE])
    C([SolydXK])
    CC([1 deep])
    CCC([2 deep])

    root --- A & B & C
    A --- AA & AB 
    C --- CC --- CCC
```

Playing with [chaining of links](https://mermaid.js.org/syntax/flowchart.html#chaining-of-links) is important and can be quite useful.
